CONTRIBUTING.md

```diff
@@ -100,6 +100,14 @@ pytest
 pytest tests/${TARGET_TEST_FILE_NAME}
 ```
 
+## Documentation
```

Comment:
There is a `### Documentation` section above, explaining `make html`, etc. How about keeping this content closer to `make html` since they're related? I'm also thinking that we could move the entire above section into this one. We don't need two.